test/csharp/project.json

```diff
@@ -15,9 +15,8 @@
 
     "frameworks": {
         "netcoreapp1.1": {
-            "imports": [ "dnxcore50", "portable-net45+win8" ],
             "dependencies": {
-                "xunit": "2.2.0-beta4-build3444",
+                "xunit": "2.2.0",
```

Comment:
Good catch, thank you! These project.json were revived when I was resolving the conflicts.